app/app_test.go

```diff
@@ -32,7 +33,12 @@ func TestGaiaApp_BlockedModuleAccountAddrs(t *testing.T) {
 	)
 	blockedAddrs := app.BlockedModuleAccountAddrs()
 
-	// TODO: Blocked on updating to v0.46.x
-	// require.NotContains(t, blockedAddrs, authtypes.NewModuleAddress(grouptypes.ModuleName).String())
 	require.NotContains(t, blockedAddrs, authtypes.NewModuleAddress(govtypes.ModuleName).String())
 }
+
+func TestGaiaApp_Export(t *testing.T) {
+	app := gaiahelpers.Setup(t, false, 1)
```

Comment:
The `isCheckTx bool` and  `invCheckPeriod uint` parameter in the `gaiahelpers.Setup` is not used; maybe we can remove them and make the setup function more simple: `app := gaiahelpers.Setup(t)`